backend/app/assets/javascripts/spree/backend/templates/orders/line_item.hbs

```diff
@@ -4,7 +4,11 @@
   </td>
 {{else}}
   <td class="line-item-image">
-    <img src="{{ image.mini_url }}">
+    {{#if image }}
+      <img src="{{ image.mini_url }}">
+    {{else}}
+      <span class="image-placeholder mini"></span>
+    {{/if}}
```

Comment:
Not sure about that because we need to do the lookup of `mini_url`/`small_url`, but maybe a helper. I'll investigate.